src/server/llmQueries.ts

```diff
@@ -49,145 +61,183 @@ type LanguageCode = "nn" | "nb" | "en";
+          Your task is to suggest a better phrasing of the text in the <excerpt> tag in EN.
+          The content in <draft> tag can be used as context, but should not be reused as part of the suggested rephrasing.
           The suggested rephrasing is to be written inside a <answer>-tag.
-          If you can not deliver the answer in the given format <answer>, the first line in the response should be <ERROR>-tag.
+          ${DEBUG_INSTRUCTION.en}
+          ${ERROR_INSTRUCTION.en}         
         </Instructions>`,
-  nn: (text: string, excerpt: string) => `<excerpt>${excerpt}</excerpt>
+  nn: (text: string, excerpt: string) => `
+        <excerpt>${excerpt}</excerpt>
         <draft>${text}</draft>
         <Instructions>
-          Du har lang erfaring frå utdanningssektoren. Du er spesialist i å reinskrive fagartiklar for å gjere det tekstlige innhaldet betre, med fokus på klarleik og lesbarheit.
-          Du har fått i oppgåve å foreslå ein betre formulering av teksten i <excerpt> på NN. Innhaldet i <draft> kan nyttast som kontekst, men skal ikkje gjenbrukast som del av den føreslåtte omformuleringa.
-          Du skal skrive refleksjonar på kvifor du har valt å inkludere dei punkta du har valt først.
+          Du har lang erfaring frå utdanningssektoren.
+          Du er spesialist i å reinskrive fagartiklar for å gjere det tekstlige innhaldet betre, med fokus på klarleik og lesbarheit.
+          Du har fått i oppgåve å foreslå ein betre formulering av teksten i <excerpt> taggen på NN.
+          Innhaldet i <draft> taggen kan nyttast som kontekst, men skal ikkje gjenbrukast som del av den føreslåtte omformuleringa.
           Forslaget til en betre formulering skal være skrive i ein <answer>-tag.
-          Om du ikkje klarar å levere svaret på formatet <answer> som er spesifisert skal fyrste linje i responsen være <ERROR>-tag.
+          ${DEBUG_INSTRUCTION.nn}
+          ${ERROR_INSTRUCTION.nn}
         </Instructions>`,
 };
 
 type AlttextQuery = Record<LanguageCode, string>;
 
 const ALT_TEXT_QUERY: AlttextQuery = {
   nb: `<Instructions>
-          Du har lang erfaring fra utdanningssektoren. Du er spesialist i å skrive alternative tekster for bilder for å gjøre dem tilgjengelige for alle elever.
-          Du har fått som oppdrag å skrive en alternativ tekst for bildet vedlagt på NB. Beskrivelsen skal være på maks 125 tegn.
-          Svaret skal leveres med refleksjoner på hvorfor du har valgt å inkludere de punktene du har valgt først, så selve alternativteksten skrevet i en <answer>-tag.
-          Om du ikke klarer å levere svaret på formatet <answer> som er spesifisert skal første linje i responsen være <ERROR>-tag.
+          Du har lang erfaring fra utdanningssektoren. 
+          Du er spesialist i å skrive alternative tekster for bilder for å gjøre dem tilgjengelige for alle elever.
+          Du har fått som oppdrag å skrive en alternativ tekst for bildet vedlagt på NB. 
```

Comment:
`Du har fått som oppdrag å skrive en alternativ tekst for det vedlagte bildet. Teksten skal være på NB.`